### PR TITLE
Cleanup webpack console build messages

### PR DESCRIPTION
**Pull request by iethree**
## Description

Doesn't print webpack build messages to the browser console anymore. Every HMR would cause several screens of build messages to appear, which, if you were looking for some application output in the console, made the console output hard to parse.

Before | After
--- | ---
Screen Shot 2022-12-16 at 2 22 34 PM | Screen Shot 2022-12-16 at 2 21 34 PM

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 /* eslint-env node */
 /* eslint-disable import/no-commonjs */
+/* eslint-disable import/order */
```
Comment (iethree) [inline webpack.config.js:3]: eslint really wanted to reorder some imports here in ways that I'm not sure is safe.

**Review by ranquild — APPROVED** [review]
